### PR TITLE
Add Splunk On-Call information

### PR DESCRIPTION
**Pull request by pauldoomgov**
Preparing to move from OpsGenie to Splunk On-Call.

### CODE REVIEW

File: _articles/platform-splunk-oncall.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Purpose
+
+Splunk On-Call provides on-call rotation scheduling and reliable alerting for
+both automated and human generated events.  It is our primary emergency
+notification system.
+
+## Getting an Account and Logging In
+
+Access can be provided for any team member with an on-call or emergency notification
+need.  Ask in #login-platform-support and tag @login-platform-help if you need
```
Comment (zachmargolis) [inline _articles/platform-splunk-oncall.md:18]:
```suggestion
need.  Ask in `#login-platform-support` and tag `@login-platform-help` if you need
```

File: _articles/appdev-oncall-guide.md [hunk 2]
```diff
@@ -11,14 +11,14 @@ subcategory: Oncall
...
+1. Access to Splunk On-Call
 2. Deploy access to production
 3. SSM access to production
 4. Join #identity-situation channel and #login-partnerships
 
-## OpsGenie Team & Rotations
+## Splunk On-Call Team & Rotations
 
-For OpsGenie access, ask Mo and the devops team.
+(See
```
Comment (zachmargolis) [inline _articles/appdev-oncall-guide.md:21]: incomplete sentence/paragraph?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/appdev-oncall-guide.md (hunk 2 above)
Comment (mitchellhenke) [reply _articles/appdev-oncall-guide.md:21]: Guessing it's supposed to link to `_articles/platform-splunk-oncall`?

File: _articles/platform-splunk-oncall.md (hunk 1 above)
Comment (pauldoomgov) [reply _articles/platform-splunk-oncall.md]: Done

File: _articles/appdev-oncall-guide.md (hunk 2 above)
Comment (pauldoomgov) [reply _articles/appdev-oncall-guide.md]: That page should be complete now.

**Pull request merged**